restapi.go

```diff
@@ -3250,6 +3250,21 @@ func (s *Session) GuildScheduledEventUsers(guildID, eventID string, limit int, w
 	return
 }
 
+// GuildOnboarding returns the onboarding flow for a guild
```

Comment:
Not all our symbols have this (since it's a non-issue), but it's probably a good idea to put a period at the end.
```suggestion
// GuildOnboarding returns the onboarding flow for a guild.
```

Same goes for all other types.